### DISCUSSION TITLE
AADSTS90023: Invalid STS request.

### DISCUSSION THREAD

**Discussion by RVelasco26** [community/community#38537] [Other Feature Feedback, Questions, & Ideas]
AADSTS90023: Invalid STS request., I got this message when I sign in using my github account in microsoft.
anyone can hellp me solve this problem?

**Comment by samus-aran** [comment]
Welcome to the GitHub Community, @RVelasco26, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
